draft-ietf-ccamp-actn-poi-pluggable-usecases-gaps.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Use cases, Network Scenarios and gap analysis for Packet Optical Integration (POI) with coherent plugables under ACTN Framework"
+title: "Use cases, Network Scenarios and gap analysis for Packet Optical Integration (POI) with programmable plugables under ACTN Framework"
```

Comment:
Would use pluggables with two g as in the rest of the document

Reply:
I agree with the changes and pluggable with 2 g